integration/src/integration/java/pl/allegro/tech/hermes/integration/PublishingTest.java

```diff
@@ -406,6 +407,11 @@ public void shouldPublishWithDelayAndConsumeMessage() {
                 .withMode(SubscriptionMode.ANYCAST)
                 .build();
         operations.createSubscription(topic, subscriptionWithDelay);
+        try {
```

Comment:
I know there were problems with this test but do we really need this ? You added new configuration which should not collide with actual tests ?